app/src/main/java/org/mozilla/webmaker/activity/Play.java

```diff
@@ -44,6 +46,42 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     @Override
+    public boolean onPrepareOptionsMenu(Menu menu) {
+        if (routeParams == null) {
+            return true;
+        }
+
+        MenuItem item = menu.findItem(R.id.action_edit);
+
+        if (item == null) {
+            return true;
+        }
+
+        String mode = "";
+        JSONObject session;
+        String creator = "";
+        String user = "";
+
+        try {
+            mode = routeParams.get("mode").toString();
```

Comment:
You know that mode will always be "play" because you are in the `Play` activity.